code/modules/power/rtg.dm

```diff
@@ -103,4 +104,40 @@
 	circuit = null
 
 /obj/machinery/power/rtg/debug/RefreshParts()
-	return
+	return ..()
```

Comment:
stop making paths that just return parent

Reply:
welp

Reply:
bruh we need it to do nothing